src/components/Message.vue

```diff
@@ -54,7 +54,12 @@
 				</div>
 				<MessageHTMLBody v-if="message.hasHtmlBody" :url="htmlUrl" />
 				<MessagePlainTextBody v-else :body="message.body" :signature="message.signature" />
-				<MessageAttachments :attachments="message.attachments" />
+				<Popover v-if="message.attachments[0]" class="attachment-popover">
+					<Actions slot="trigger">
+						<ActionButton icon="icon-public icon-attachment">Attachments</ActionButton>
+					</Actions>
+					<MessageAttachments :attachments="message.attachments" />
+				</Popover>
 				<div id="reply-composer"></div>
```

Comment:
Any idea why this "reply-composer" thingy is still here? Doesn't seem to be used...

Reply:
No idea... but I didn't want to touch code which I don't know what purpose it might serve... Maybe @ChristophWurst can help us here?

Reply:
I think this can be removed in a separate PR